plugins/c8y_log_plugin/src/main.rs

```diff
@@ -154,6 +125,59 @@ async fn run(
     }
 }
 
+pub async fn process_mqtt_messages(
+    message: Message,
+    plugin_config: &LogPluginConfig,
+    mqtt_client: &mut Connection,
+    http_client: &mut JwtAuthHttpProxy,
+    config_file: &Path,
+) -> Result<(), anyhow::Error> {
+    if is_c8y_bridge_up(&message) {
+        let plugin_config = read_log_config(config_file);
+        let () = handle_dynamic_log_type_update(&plugin_config, mqtt_client).await?;
+    }
+    match message.topic.name.as_str() {
+        "tedge/health-check" | "tedge/health-check/c8y-log-plugin" => {
```

Comment:
This needs to be replaced by a `if health_check_topics.accept(&message)`

Reply:
Fixed it `if health_check_topics("c8y-log-plugin").accept(&message)`